docs/event-spec.md

```diff
@@ -50,7 +50,11 @@ vdom will:
 
 ```python
 {
-    'event_type': '{hash}_{target_name}'
+    '{eventName}': {
```

Comment:
Can you add some text in the surrounding documentation to clarify what `stopPropagation` and `preventDefault` mean?

Reply:
Yup, will add docs for that and for the new event handler decorator.

Reply:
@captainsafia added the docs:

https://github.com/nteract/vdom/pull/99/files#diff-e11a66e3327aabbe4d30a89c0469e07eR43